Makefile

```diff
@@ -654,9 +654,6 @@ update-go-version: ## Update Go version for all the components (images, CI, dev-
 	@echo "Updated go version in test scripts to $(GO_VERSION)"
 	# Update Go version in Dockerfiles.
 	$(QUIET) sed -i 's/GOLANG_VERSION=.*/GOLANG_VERSION=$(GO_VERSION)/g' contrib/backporting/Dockerfile
-	$(QUIET) sed -i 's/^go_version=.*/go_version=$(GO_IMAGE_VERSION)/g' images/scripts/update-golang-image.sh
-	$(QUIET) $(MAKE) -C images update-golang-image
-	@echo "Updated go version in image Dockerfiles to $(GO_IMAGE_VERSION)"
```

Comment:
Does this mean we no longer need to update the runtime and builder images on Go version updates?

Reply:
@tklauser please see #20850